packages/odata-service-inquirer/src/utils/index.ts

```diff
@@ -13,19 +12,6 @@ import type { ListChoiceOptions } from 'inquirer';
 
 const osVersionName = osName();
 
-/**
- * Determine if the current prompting environment is cli or a hosted extension (app studio or vscode).
- *
- * @returns the platform name and technical name
- */
-export function getHostEnvironment(): { name: string; technical: string } {
```

Comment:
Since this returns different results that the one that has been added to `fiori-generator-shared` I dont think we can replace it?

Reply:
Reverted the changes